### PR TITLE
Dokka

### PR DESCRIPTION
**Pull request by hannesa2**
Working example for javadoc with Kotlin and auto upload to https://jitpack.io

### CODE REVIEW

File: DBPrefs/src/main/java/info/dbprefs/lib/DbPreferences.kt [hunk 1]
```diff
@@ -14,6 +14,9 @@ import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.schedulers.Schedulers
 import java.lang.reflect.Type
 
+/**
+ * An encrypted alternative to SharedPreferences. It's based on secure Room/SQlite and uses sqlcipher
```
Comment (hannesa2) [inline DBPrefs/src/main/java/info/dbprefs/lib/DbPreferences.kt]: This is the only place to check, if it's working properly

**Comment by hannesa2** [comment]
It's not perfect, in **Java** it works properly

image


But **Kotlin** not 👎 

image

**Comment by originx** [comment]
https://github.com/Kotlin/dokka/issues/215

**Comment by hannesa2** [comment]
all https://github.com/Kotlin/dokka/issues/371 are pointing to https://github.com/Kotlin/dokka/issues/215

**Pull request merged**